safe/impact_functions/base.py

```diff
@@ -208,6 +234,26 @@ def actual_extent_crs(self):
         return self._actual_extent_crs
 
     @property
+    def map_canvas(self):
```

Comment:
Yes, I have though the same too. But I need to extent of the map canvas in order to compute the best extent between exposure, hazard, user extent and the map canvas extent.
To be more consistent with other properties like ```requested_extent``` and ```actual_extent```, I can use ```map_canvas_extent```. Is-it ok ?